cluster/manifests/roles/poweruser-role.yaml

```diff
@@ -0,0 +1,846 @@
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.authorization.k8s.io/aggregate-to-poweruser: "true"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: poweruser
+rules: [] # rules aggregated from other sub roles
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: poweruser-base
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-poweruser: "true"
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/attach
+  - pods/proxy
+  - pods/exec
+  - pods/portforward
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+  - deletecollection
+- apiGroups:
+  - ""
+  resources:
+  - replicationcontrollers
+  - replicationcontrollers/scale
+  - serviceaccounts
+  - services
+  - services/proxy
+  - endpoints
+  - persistentvolumeclaims
+  - configmaps
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+  - deletecollection
+- apiGroups:
+  - ""
+  resources:
+  - limitranges
+  - resourcequotas
+  - bindings
+  - events
+  - pods/status
+  - resourcequotas/status
+  - namespaces/status
+  - replicationcontrollers/status
+  - pods/log
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - impersonate
```

Comment:
why?

Reply:
Good catch, this list was generated from the official `edit` role which is default in Kubernetes. I missed this, and will remove it now after a better review of the permissions in this list.

Reply:
(It would have been blocked by our webhook though) :)